dcos_test_utils/gce.py

```
@@ -0,0 +1,174 @@
+import os
+import json
+import logging
+from googleapiclient.errors import HttpError
+from googleapiclient import discovery
+from oauth2client.client import GoogleCredentials
+from retrying import retry
+
+log = logging.getLogger(__name__)
+
+
+def catch_http_exceptions(f):
+    def handle_exception(*args, **kwargs):
+        try:
+            return f(*args, **kwargs)
+        except HttpError as e:
```

Will this hide errors that are not `404` or `409`?

---

Good call